Code/JavaWrappers/gmwrapper/src-test/org/RDKit/SubstanceGroupTests.java

```diff
@@ -0,0 +1,93 @@
+//
+// Created by Gareth Jones on 9/4/2020.
+//
+// Copyright 2020 Schrodinger, Inc
```

Comment:
Same comment about the boilerplate here

Reply:
I've added the boilerplate- I've also updated the files I created for the tautomer search.

Reply:
thanks. Sorry I missed those at the time